xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd

```diff
@@ -737,6 +737,16 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>The number of wheelchairs places on vehicles of the type. For a requirement this is the minimum needed.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="PramPlaceCapacity" type="NumberOfPassengers" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>The number of places on the vehicle that are suitable for prams.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="BicycleRackCapacity" type="NumberOfPassengers" minOccurs="0">
```

Comment:
Bicycle places are not limited to racks, e.g., there are systems with straps or just free space for any kind of luggage. Here a proposal with a different name. Note that the example would have to be adapted as well...
```suggestion
			<xsd:element name="BicycleCapacity" type="NumberOfPassengers" minOccurs="0">
```

Reply:
@trurlurl I agree but the element has that name in SIRI. So if we change it here, I would need to put a PR in SIRI As well.  @skinkie @Aurige your opinion?

Reply:
Yes, I realized it too late that we can't just change the element name. But can we change the definition / description - or is it important to only count places related to racks? If not, the description should help ensuring the interpretation we want.

Reply:
I'd rather align with SIRI